server/schema/types/create_user.js

```diff
@@ -0,0 +1,10 @@
+import {GraphQLInputObjectType, GraphQLNonNull, GraphQLString} from 'graphql';
```

Comment:
suggest we change the file name to `user_input.js`, because this is the type for generic user, we can also use it for `update_user`